build-aux/config.guess

```diff
@@ -1483,8 +1482,8 @@ EOF
     i*86:rdos:*:*)
 	echo "$UNAME_MACHINE"-pc-rdos
 	exit ;;
-    i*86:AROS:*:*)
-	echo "$UNAME_MACHINE"-pc-aros
+    *:AROS:*:*)
+	echo "$UNAME_MACHINE"-unknown-aros
```

Comment:
Why did we need to change this? Did it produce a new build error once we bumped libc?

Reply:
This is not related to this pr, but this always changes when I build emacs-ng. Do these files not change when you run the build ?

Reply:
I dropped the changes.